### PR TITLE
Pass down information needed for d2l-sequences to redirect to content…

### PR DESCRIPTION
**Pull request by qwangd2l**
… service.

~~*Do not merge until d2l-sequences' changes are released, and then bump the version referenced in this repo.~~ It is good to merge now since d2l-sequences has been updated and released. Package-lock.json in this repo is updated to use this new release.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   },
   "homepage": "https://github.com/Brightspace/sequence-viewer",
   "name": "d2l-sequence-viewer",
-  "version": "1.3.19",
+  "version": "1.3.20",
```
Comment (ChrisKraljevicD2L) [inline package.json]: This is new functionality, the new version should be `1.4.0`

File: package-lock.json [hunk 2]
```diff
@@ -2703,6 +2704,22 @@
       "requires": {
         "@brightspace-ui/core": "^1",
         "@polymer/polymer": "^3.0.0"
+      },
+      "dependencies": {
```
Comment (ChrisKraljevicD2L) [inline package-lock.json]: This is often a sign that dependencies weren't flattened properly while `npm i`'ing. I recommend deleting `node_modules` and `package-lock.json` and running `npm i` again.

Comment (qwangd2l) [reply package-lock.json]: Interesting, I'll try it again.

**Comment by ChrisKraljevicD2L** [comment]
Ah, I screwed something up. I had to rename a function I added to `core` that's getting pulled in with this update. I'm gonna fix that and I'll let you know when it's cool to merge master into this branch and get this merged :+1:

**Comment by ChrisKraljevicD2L** [comment]
@qwangd2l I'm so sorry this took so long but it's cool to merge master now

**Pull request merged**